doc/usage/restructuredtext/basics.rst

```diff
@@ -208,11 +208,31 @@ Hyperlinks
 External links
 ~~~~~~~~~~~~~~
 
-Use ```Link text <https://domain.invalid/>`_`` for inline web links.  If the
-link text should be the web address, you don't need special markup at all, the
-parser finds links and mail addresses in ordinary text.
+URLs and email addresses in text are automatically linked an do no need explicit
```

Comment:
typo(s) "an do no need"

Reply:
Hello @timhoffm I do not know why but this is still standing as far as I can see, so I temporarily "unresolve" this conversation...